Dockerfile

```diff
@@ -1,5 +1,5 @@
 FROM ruby:2.5.3-alpine
-ARG BUNDLE_INSTALL_CMD
+ARG BUNDLE_INSTALL_CMD=bundle
```

Comment:
thanks, been caught by this a few times when sidestepping `make`

Reply:
<3